src/onnx/parse_resize.cpp

```diff
@@ -375,8 +354,9 @@ struct parse_resize : op_parser<parse_resize>
                 }
             });
 
-            auto ind = calc_neighbor_points(
-                vvv_ind, 0, std::vector<std::vector<std::size_t>>(out_elements), in_s);
+            std::vector<int> ind;
```

Comment:
(as mentioned above, let `ind` need not be an input_parameter)

Reply:
I'll revise to the original way by returning a vector directly.